src/main/webapp/WEB-INF/jsp/editDisplayPreferences.jsp

```diff
@@ -51,20 +51,20 @@
                                         <td>
                                             <c:choose>
                                                 <c:when test="${ts.topic.subscriptionMethod == 1}">
-                                                    <input type="checkbox" disabled="disabled" checked="checked" value="true" name="subscribed_${status.index}"/>
+                                                    <input role="checkbox" aria-labelledby="SubscriptionName" type="checkbox" disabled="disabled" checked="checked" value="true" name="subscribed_${status.index}" aria-checked="true"/>
```

Comment:
:eye_speech_bubble:  `<input type="checkbox" role="checkbox" ...>`
Most screen readers use `type` as a default value for `role`.
In this case is the `role` needed?